browser/css/jsdialogs.css

```diff
@@ -1758,17 +1755,30 @@ input[type='number']:hover::-webkit-outer-spin-button {
 }
 
 /* toolbuttons */
-
-.jsdialog:not(.menubutton) > .ui-content.unobutton {
-	width: var(--btn-size);
-}
-
 .jsdialog .ui-content.unobutton {
-	height: var(--btn-size);
-	padding: 0px;
+	display: flex;
+	flex-direction: column;
+	padding: 3px;
 	vertical-align: middle;
 	background-color: transparent;
 	border: none;
+	position: relative;
+}
+
+.unoarrow::after,
+.jsdialog .ui-content.unobutton::after {
```

Comment:
I extended a little bit the click zone so that the border of the button could be clickable